tools/packaging/static-build/cache_components.sh

```diff
@@ -103,8 +113,14 @@ cache_rootfs_artifacts() {
 			root_hash_vanilla=""
 			root_hash_tdx="${repo_root_dir}/tools/osbuilder/root_hash_tdx.txt"
 		fi
+		if [ "${TEE}" == "sev" ]; then
+			rootfs_tarball_name="kata-static-cc-sev-rootfs-initrd.tar.xz"
+			aa_kbc="online_sev_kbc"
+			image_type="initrd"
+			initramfs_version="-$(get_initramfs_image_name)"
```

Comment:
Same comment here about the initramfs_version being misleading and we actually needing the last modification of the builder image.

Reply:
Changed the variable name.